jobs/validate-tigera-secure-ee/Jenkinsfile

```diff
@@ -0,0 +1,82 @@
+@Library('juju-pipeline@master') _
+
+def juju_model = String.format("%s-%s", params.model, uuid())
+
+pipeline {
+    agent {
+        label 'runner-amd64'
+    }
+    /* XXX: Global $PATH setting doesn't translate properly in pipelines
+     https://stackoverflow.com/questions/43987005/jenkins-does-not-recognize-command-sh
+     */
+    environment {
+        PATH = "/snap/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin"
+    }
+    options {
+        ansiColor('xterm')
+        timestamps()
+    }
+    stages {
+        stage('Deploy: K8s') {
+            options {
+                timeout(time: 1, unit: 'HOURS')
+            }
+            steps {
+                sh "juju kill-controller -y ${params.controller} || true"
+                dir('jobs') {
+                    script {
+                        sh "CONTROLLER=${params.controller} tox -e py36 -- python3 integration/tigera/bootstrap_aws_single_subnet.py"
+                    }
+                }
+                setStartTime()
+                deployCDK(controller: params.controller,
+                          model: juju_model,
+                          bundle: "cs:~containers/${params.bundle}",
+                          version_overlay: params.overlay,
+                          bundle_channel: params.bundle_channel,
+                          disable_wait: true)
+                license_key_base64 = credentials('tigera-secure-ee-license-key').bytes.encodeBase64().toString()
+                registry_credentials_base64 = credentials('tigera-private-registry-credentials').bytes.encodeBase64().toString()
+                sh "juju config -m ${params.controller}:${params.model} tigera-secure-ee license-key=${license_key_base64} registry-credentials=${registry_credentials_base64}"
+                dir('jobs') {
+                    sh "tox -e py36 -- python3 integration/tigera/disable_source_dest_check.py -m ${params.controller}:${params.model}"
+                }
+                sh "juju-wait -e ${params.controller}:${params.model} -w"
+            }
+        }
+
+        stage('Validate') {
+            options {
+                timeout(time: 2, unit: 'HOURS')
+            }
+
+            steps {
+                dir('jobs') {
+                    sh "CONTROLLER=${params.controller} MODEL=${juju_model} tox -e py36 -- pytest -v -s --junit-xml=validate.xml integration/test_cdk.py::test_validate"
+                }
+            }
+        }
+    }
+    post {
+        failure {
+            setFail()
+        }
+        success {
+            setPass()
+        }
+        always {
+            setEndTime()
+            collectDebug(params.controller,
+                         juju_model)
+
+            dir('jobs') {
```

Comment:
Not a blocker, but it may be worth putting this in the `cleanup {}` stanza

Reply:
Ah good catch, thanks! Opened a new PR for this: https://github.com/juju-solutions/kubernetes-jenkins/pull/325